lib/generators/administrate/dashboard/dashboard_generator.rb

```diff
@@ -16,7 +15,9 @@ class DashboardGenerator < Rails::Generators::NamedBase
       }
 
       ATTRIBUTE_OPTIONS_MAPPING = {
-        enum: { searchable: false },
+        # procs must be defined in one line!
+        enum: {  searchable: false,
+                 collection: ->(field) { field.resource.class.send(field.attribute.to_s.pluralize).keys } },
```

Comment:
It's a little thing, but I wonder if we can use the `lamdba do; end` syntax here to achieve the same goal?

Reply:
It all depends on how good the parser will be, because I'm fetching raw file lines there and check them using regexp. We'll see how this idea works out and I'll try to implement parser for all possible syntaxes if it's the way to go.